src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -2217,6 +2254,12 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
             let data    = visualization::Data::from(content);
             inputs.set_visualization_data.emit((*node_id,data));
         }
+
+        let sample_doc_generator = MockDocGenerator::default();
+        let data                 = sample_doc_generator.generate_data();
+        let content              = serde_json::to_value(data).unwrap();
```

Comment:
unwrap can panic. We do not allow such unwraps in the application. Please handle it safely.